README.md

```diff
@@ -190,10 +190,11 @@ Merges the Dublin Core metadata into the hOCR file by encoding the data in its h
 ### hocr-pdf
 
 ```
-Usage: hocr-pdf <imgdir>
+hocr-pdf <imgdir> > out.pdf
+hocr-pdf <imgdir> --savefile out.pdf
```

Comment:
The help text shows a different order:

    usage: hocr-pdf [-h] [--savefile SAVEFILE] imgdir

    Create a searchable PDF from a pile of hOCR and JPEG

    positional arguments:
      imgdir               directory with the hOCR and JPEG files (corresponding
                           JPEG and hOCR files have to have the same name with
                           their respective file ending)

    optional arguments:
      -h, --help           show this help message and exit
      --savefile SAVEFILE  Save to this file instead of outputting to stdout